app/experimenter/experiments/models/nimbus.py

```diff
@@ -130,28 +130,11 @@ class Meta:
         verbose_name_plural = "Nimbus Experiments"
 
     class Filters:
-        IS_LAUNCH_QUEUED = Q(
-            status=NimbusConstants.Status.DRAFT,
-            publish_status=NimbusConstants.PublishStatus.APPROVED,
-        )
-        IS_LAUNCHING = Q(
-            status=NimbusConstants.Status.DRAFT,
-            publish_status=NimbusConstants.PublishStatus.WAITING,
-        )
-        IS_PAUSE_QUEUED = Q(
-            status=NimbusConstants.Status.LIVE,
-            is_paused=False,
-        )
-        IS_END_QUEUED = Q(
-            status=NimbusConstants.Status.LIVE,
-            publish_status=NimbusConstants.PublishStatus.APPROVED,
-            is_end_requested=True,
-        )
-        IS_ENDING = Q(
-            status=NimbusConstants.Status.LIVE,
-            publish_status=NimbusConstants.PublishStatus.WAITING,
-            is_end_requested=True,
-        )
+        IS_LAUNCH_QUEUED = Q(**NimbusConstants.LifecycleStates.DRAFT_APPROVED.value)
+        IS_LAUNCHING = Q(**NimbusConstants.LifecycleStates.DRAFT_WAITING.value)
+        IS_PAUSE_QUEUED = Q(**NimbusConstants.LifecycleStates.PAUSE_QUEUED.value)
+        IS_END_QUEUED = Q(**NimbusConstants.LifecycleStates.LIVE_APPROVED_ENDING.value)
+        IS_ENDING = Q(**NimbusConstants.LifecycleStates.LIVE_WAITING_ENDING.value)
```

Comment:
This is even more interesting!!!  Makes total sense, I like that you can define filters in relation to lifecycles.  So interesting.